lib/events.js

```diff
@@ -1413,3 +1485,161 @@ async function _getStartHash(
   const eventHash = await getStartHash({creatorId, targetGeneration});
   return {eventHash, truncated: true};
 }
+
+/*
+  LOOKING FOR A HOME
```

Comment:
Needs some attention (either a new home, or a decision that it doesn't need one with whatever reworking is necessary).

Reply:
Leaving this for now by consensus.